test/functionalTest/cases/000_ipv6_support/ipv4_ipv6_both.test

```diff
@@ -124,6 +124,7 @@ orionCurl --url "$url" --payload "$payload" --host "[::1]" -g 2> /dev/null
 
 echo "6: ++++++++++++++++++++"
 #Get accumulated data
+valgrindSleep 2
 curl -g [::1]:${LISTENER2_PORT}/dump -s -S 2> /dev/null
```

Comment:
It's a pitty cannot use accumulatorDump function for this due to the weird IPv6 syntax... Maybe a parameter could be used?

```
accumulatorDump IPv6
```

Not sure, but I think I remember we have used that technique some time (maybe in accumulatorStart function?)